tools/metatoenv.py

```diff
@@ -136,13 +152,31 @@ def _jinja_filter(dependencies, platform):
                                        "unmatched square brackets or closing bracket "
                                        "found before opening bracket: {}".format(key))
                 selector = key[left:right + 1]
-                if selector.startswith('[not'):
-                    if (platform in selector) or (selector.replace(
-                            '[not', '')[:-1].strip() in platform):
+                raw_selector = selector.lstrip('[ ').rstrip(' ]').replace(
+                    'python', 'py').replace('64', '').replace('32', '')
+
+                if raw_selector.startswith('py'):
+                    # Check for python version
+                    select_ver = raw_selector.replace('>',
+                                                      '=').replace('<',
+                                                                   '=').split('=')[-1]
```

Comment:
Could do this in a single call:
```suggestion
                    select_ver = re.split(r'<|>|\=', raw_selector)[-1]
```